MykhailoTymoshchuk/HW7/task3.py

```diff
@@ -0,0 +1,28 @@
+# def calculateCharacter (input_str):
+"""Create a simple function
+char_count = {} for short time , pure dict to wich we add a char after each iterations """
+#     char_count = {}
+    
+#     for char in input_str:
+#         if char in char_count:
+#             char_count[char] += 1
+#         else:
+#             char_count[char] = 1
+            
+#     return char_count
+"""Contidion with input sting "Hello"
+ouput_dict = new dict with counted char from the cicle condition above """
+# input_str = "hello"
+# output_dict = calculateCharacter(input_str)
+# print(output_dict)
+
```

Comment:
good idea
look at this implementation
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result